### PR TITLE
feat: add My spaces page

### PR DESCRIPTION
**Pull request by j0hnfl0w**
Closes: https://github.com/snapshot-labs/sx-ui/issues/467

### CODE REVIEW

File: src/components/SpaceItem.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<script lang="ts" setup>
+import { Space } from '@/types';
+import { useSpacesStore } from '@/stores/spaces';
```
Comment (Sekhmet) [inline src/components/SpaceItem.vue:3]:
```suggestion
import { useSpacesStore } from '@/stores/spaces';
import { Space } from '@/types';
```

nit. we tend to put `@/types` imports at the bottom (here and in other files in this PR).

**Review by Sekhmet — APPROVED** [review]
tACK

File: src/views/Settings/Spaces.vue [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <Container class="max-w-screen-md pt-5">
+      <h2 class="mb-4 mono !text-xl" v-text="'My spaces'" />
+      <UiLoading v-if="!loaded" class="block mb-2" />
+      <div v-if="loaded && !spaces.length" class="py-3 flex items-center text-skin-link">
+        <IH-exclamation-circle class="inline-block mr-2" />
+        <span v-text="'There are no spaces here.'" />
+      </div>
+      <div v-else-if="loaded" class="max-w-screen-md">
+        <div class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-3 mb-3">
+          <SpaceItem v-for="space in spaces" :key="space.id" :space="space" />
```
Comment (bonustrack) [inline src/views/Settings/Spaces.vue:23]: Shouldn't we use also `SpaceItem` in the explore page?

**Review by bonustrack — APPROVED** [review]
tACK

**Pull request merged**